aries-site/src/layouts/main/Layout.js

```diff
@@ -64,7 +64,10 @@ export const Layout = ({ children, title }) => {
               }}
             >
               <Box fill="vertical">
-                <SideBar items={SideBarItemList[selectedNav]} />
+                <SideBar
+                  items={SideBarItemList[selectedNav]}
+                  prefix={selectedNav}
+                />
```

Comment:
So, since this layout now always has a sidebar, I think we will need to create two new issues:
1) We will need a layout for the "landing page" which does not have a sidebar
2) Hide sidebar at the same breakpoint Nav goes from desktop to mobile

Do those items sound correct?

Reply:
Agreed on both of those items, @halocline. We could make a `LandingPage` within the layouts folder. For the sidebar, the breakpoint should be at `small`

Reply:
Sounds good to me.
IMO we will need a separate layout for the landing page, but let's see how the design looks like first and go from there.

Reply:
Yes, I agree that it will be a separate layout. I meant it could be a file inside the layouts directory. maybe: `layouts/main/LandingPageLayout.js`